soiree/app/controllers/group/playlists_controller.rb

```diff
@@ -16,17 +16,21 @@ def create
 
   def destroy
     playlist = Playlist.find(params[:id])
-
-    delete_group_redirect(playlist)
+    delete_group_invites(playlist)
+    redirect_to dashboard_path
   end
 
   private
 
-  def delete_group_redirect(playlist)
-    playlist.group_playlists.destroy_all
-    playlist.playlist_songs.destroy_all
-    playlist.destroy
-    redirect_to dashboard_path
+  def delete_group_invites(playlist)
+    group = playlist.groups[0]
```

Comment:
Does each playlist always only have one group? If so, it seems like you could access this without accessing an array and grabbing the first one - you could just grab its one group. If not, how do you know that the group you're going for is always the first one in the list?